src/domain/event/EventOccurrence.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { eventQuery_event_occurrences_edges_node as OccurrencesEdgeNode } from '../api/generatedTypes/eventQuery';
+import Button from '../../common/components/button/Button';
+import { formatTime, newMoment } from '../../common/time/utils';
+import styles from './eventOccurrence.module.scss';
+
+interface EventOccurrenceProps {
+  occurrence: OccurrencesEdgeNode;
+}
+
+const EventOccurrence: React.FunctionComponent<EventOccurrenceProps> = ({
+  occurrence,
+}) => {
+  const { t } = useTranslation();
+
+  // TODO: Ensure that date shows in correct locale.
+  const date = formatTime(newMoment(occurrence.time), 'dd l');
+  const time = formatTime(newMoment(occurrence.time), 'hh:mm');
+
+  return (
+    <tr className={styles.occurrence}>
+      <td>{date}</td>
+      <td>{time}</td>
+      <td>
+        {occurrence.venue.name} {occurrence.venue.description}
```

Comment:
Should venue's description not be here?

Reply:
I actually don't know what should be there, we'll find out pretty soon..